sample/src/main/java/dev/arkbuilders/sample/storage/StorageDemoFragment.kt

```diff
@@ -21,8 +22,8 @@ class StorageDemoFragment: DialogFragment() {
     private val TAG = StorageDemoFragment::class.java.name
 
     private lateinit var binding: FragmentStorageDemoBinding
-    private val map by lazy { mutableMapOf<String, String>() }
     private var workingDir: String = "/"
+    private var storage: FileStorage? = null
```

Comment:
Will everything work if we specific implementation `FileStorage` with interface `BaseStorage`?

Reply:
BaseStorage doesn't have bindings yet :thinking:

Should it have?

Reply:
I didn't know 😁 In the end yes, but we can test with specific bindings first.